style.css

```diff
@@ -188,375 +202,472 @@ section {
-  font-weight: 700;
   font-style: normal;
+  font-weight: 700;
   font-size: 40px;
+  line-height: 52px;
   display: flex;
   align-items: center;
   text-align: center;
-  margin-top: 150px;
-  margin-bottom: 12px;
   color: #fff;
-  justify-content: center;
 }
 
 .wordings {
-  width: calc(100%);
   font-family: "Poppins", sans-serif;
-  font-weight: 400;
   font-style: normal;
-  font-size: 28px;
-  display: flex;
-  align-items: center;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
   text-align: center;
   color: #ebebff;
-  justify-content: center;
 }
 
-.contact-form {
-  width: 100%;
-  margin-bottom: 45px;
+.background-image {
+  background: url(Images/dt-background-shapes.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: contain;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  padding: 114px 40px;
+}
+
+.scontainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 0;
 }
 
 .input-control input {
-  height: 48px;
-  width: 100%;
   border-radius: 8px;
-  padding: 15px 38px 15px 16px;
-  margin: 10px 0;
+  border: none;
+  outline: none;
+  padding: 15px 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  width: 100%;
 }
 
-#message {
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.input-control textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 100%;
   height: 170px;
+  padding: 15px 16px;
+  line-height: 24px;
+  font-size: 15px;
+  font-weight: 400;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.input-control textarea::placeholder {
+  font-size: 15px;
+  font-weight: 400;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
   width: 100%;
-  border-radius: 8px;
-  margin-top: 10px;
-  padding: 15px 38px 15px 16px;
 }
 
-#btn {
-  width: 140px;
-  height: 48px;
+input[type="submit"] {
   float: left;
+  height: 48px;
+  color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 5px;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  cursor: pointer;
+  background-color: #fff;
+  width: fit-content;
 }
 
-.footer-line {
-  width: 135px;
-  height: 5px;
-  bottom: 8px;
-  background: #0d151e;
-  border-radius: 10px;
-  margin-left: 32%;
-  margin-right: 32%;
-  margin-bottom: 24px;
+input[type="submit"]:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+@media (min-width: 768px) {
+  .heading {
+    padding: 0 20px;
+  }
+
+  .head {
+    background-color: #f7f7f9;
+  }
+
+  .heading .menu-icon {
+    display: none;
+  }
+
+  .dmenu {
+    display: flex;
+    flex-direction: row;
+    line-height: 30px;
+  }
```

Comment:
Great job @Rn486 🤝

- [ ] [Optional] You may please add a margin to the right of your menu items so that your design can replicate the design provided in the Figma template.